packages/extensions/modelerfour/test/modeler/modelerfour.requests.test.ts

```diff
@@ -234,4 +234,70 @@ describe("Modelerfour.Request", () => {
       expect(parameters[1].protocol).toEqual({ http: { in: "header" } });
     });
   });
+
+  describe("final-result-schema in lro options", () => {
+    it("creates a 200 response from final-result-schema if one is not present", async () => {
+      const productSchema = {
+        type: "object",
+        properties: {
+          id: {
+            type: "string",
+          },
+        },
+      };
+
+      const oldSpec: oai3.Model = createTestSpec();
+      addOperation(oldSpec, "/products/{id}", {
+        put: {
+          operationId: "product_create",
+          description: "Create a Product",
+          "x-ms-long-running-operation": true,
+          "x-ms-long-running-operation-options": {
+            "final-state-via": "location",
+          },
+          responses: {
+            "200": {
+              description: "Success",
+              content: {
+                "application/json": {
+                  schema: { $ref: "#/components/schemas/Product" },
+                },
+              },
+            },
+            "202": {
+              description: "Accepted",
+            },
+          },
+        },
+      });
+      addSchema(oldSpec, "Product", productSchema);
+
+      const newSpec: oai3.Model = createTestSpec();
+      addOperation(newSpec, "/products/{id}", {
+        put: {
+          operationId: "product_create",
+          description: "Create a Product",
+          "x-ms-long-running-operation": true,
+          "x-ms-long-running-operation-options": {
+            "final-state-via": "location",
+            "final-state-schema": "#/components/schemas/Product",
```

Comment:
Can we add a test that validates we still error out if there is no 200 AND there is no `final-state-schema`?
Maybe another one that validates we get an error if we both are set?

Reply:
Good suggestions @m-nash! I have added these two tests.

But note that there is no error for an LRO without a 200 response -- modeler4 allows this. The test now confirms this behavior.